### PR TITLE
Add magic-script icon for Lumin app.

### PR DESCRIPTION
**Pull request by pleszkiewicz**
@panwrona This PR shows how to integrate portal icon with lumin app - you may use it in CLI template project.
mxs_icon1

mxs_icon2

### CODE REVIEW

File: lumin/res/Icon/model/model.kmat [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      1,
+      1,
+      1
+    ],
+    "blendmode": "Opaque"
+  },
+  "materials": [
+    {
+      "shaderName": "UnlitTextured",
+      "name": "org_magicscript_catalog_model",
```
Comment (pleszkiewicz) [inline lumin/res/Icon/model/model.kmat:14]: @panwrona please note the format of the "name"

File: lumin/res/Icon/portal/portal.kmat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      1,
+      1,
+      1
+    ],
+    "blendmode": "Opaque"
+  },
+  "materials": [
+    {
+      "shaderName": "UnlitTextured",
+      "name": "org_magicscript_catalog_portal",
```
Comment (pleszkiewicz) [inline lumin/res/Icon/portal/portal.kmat:14]: @panwrona please note the format of the "name"

Comment (panwrona) [reply lumin/res/Icon/portal/portal.kmat:14]: Thanks for information, with the MagicScript CLI the name should be created dynamically if the icon is there right?

File: lumin/app.package [hunk 3]
```diff
@@ -1,5 +1,6 @@
 DATAS = "digest.sha512.signed" : "." \
         "bin/" : "bin/" \
+        "res/" : "res/" \
```
Comment (panwrona) [inline lumin/app.package:3]: This icon will be used internally by Lumin or by every platform?

**Review by panwrona — APPROVED** [review]
Looks good, thanks for the info about naming, please take a look at my comments though :)

File: lumin/app.package (hunk 3 above)
Comment (pleszkiewicz) [reply lumin/app.package:3]: This is 3d icon (PlatformIcon) so we cannot reuse it in iOS/Android. But we may provide default icons for react-native as well.

Comment (pleszkiewicz) [reply lumin/app.package:3]: This is portal icon (= 3d icon) so only by Lumin.

**Pull request merged**